kahuna/public/js/components/gr-add-preset-label/gr-add-preset-label.css

```diff
@@ -0,0 +1,31 @@
+gr-add-preset-label{
+    display: flex;
+    height: 25px;
+    padding: 5px;
+}
+
+gr-add-preset-label:hover {
+    color: white;
+}
+
+gr-add-preset-label .small {
```

Comment:
Is the class `.small` used anywhere else? If not could we make it a `--small` modifier.